wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -730,6 +730,15 @@ public final class SessionManager: NSObject, SessionManagerType {
             if deleteAccount {
                 self?.deleteAccountData(for: account)
             }
+
+            // Clear tmp directory when the user logout from the session.
+            guard let tmpDirectoryPath = URL(string: NSTemporaryDirectory()) else { return }
+            let manager = FileManager.default
+            try? manager
+                .contentsOfDirectory(at: tmpDirectoryPath, includingPropertiesForKeys: nil, options: .skipsSubdirectoryDescendants)
+                .forEach { file in
+                    try? manager.removeItem(atPath: file.path)
+                }
```

Comment:
I suggest to move this into a new method `deleteTemporaryData()`.

Reply:
updated.